test/log-generators/cronjobs/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Create a CronJob that run every minute, creating an Apache log file 1024 lines long.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
Is this line used to locate the  directory where install.sh is no matter from where this is ran?

Maybe we could add a comment stating that.
I didn't thought about this but we could take advantage of this approach in future  scripts.